.github/workflows/docker_update_perftool.yml

```diff
@@ -0,0 +1,78 @@
+# Anyone with write permissions to the antrea-io/antrea Github repository can
+# trigger this workflow manually, but please check with a maintainer first. The
+# workflow will build and push the antrea/perftool and antrea/netperf image,
+# with multi-platform support.

```

Comment:
confused by the differences between the 2 workflows: they both have the same top-level comment and they both push tags for `antrea/perftool` and `antrea/netperf`